js/index.js

```diff
@@ -40,3 +40,62 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let logo2 = document.getElementById("cta-img");
+logo2.setAttribute('src', siteContent["cta"]["img-src"])
```

Comment:
Yea was trying it out with the way the example showed, but with dot notation it would look something like this 

```logo.src = siteContent.nav['img-src'];```